src/action/common/configure_init_service.rs

```diff
@@ -311,20 +317,6 @@ impl Action for ConfigureInitService {
                     })
                     .map_err(Self::error)?;
 
-                Self::check_if_systemd_unit_exists(SOCKET_SRC, SOCKET_DEST)
```

Comment:
Oh? Why was this removed?

Reply:
Correct me if I'm going insane, but it's a repeat of the above block and likely a bad merge?

Reply:
This block was for `SOCKET_{SRC,DEST}`, while the block above is for `SERVICE_{SRC,DEST}`, so I don't think it's a repeat? But I could be wrong.

Reply:
Oh my god.